integration-tests/build.gradle

```diff
@@ -47,7 +47,6 @@ dependencies {
     androidTestImplementation 'androidx.test:runner:1.4.0'
     androidTestImplementation 'androidx.test:rules:1.4.0'
     androidTestImplementation 'androidx.test.ext:junit-ktx:1.1.3'
-    androidTestImplementation 'org.assertj:assertj-core:3.22.0'
```

Comment:
Hmm interesting that R8 is failing with this, maybe we could also try to update to the latest assertj version to see if they fixed it. Haven't found the CHANGELOG though, so might not fix it...

Reply:
Oh AS wasn't offering me to upgrade to a more recent version so I assumed this was the latest one, let me try with 3.24.2

Reply:
Ok, tried and it still failed [`22740ec` (#998)](https://github.com/RevenueCat/purchases-android/pull/998/commits/22740eca8d337a9a5f640682b54c1132e3b04ba2)